### ISSUE TITLE
Release non-prerelease version to nuget

### ISSUE THREAD

**Issue by cmeeren** [fsprojects/FSharpPlus#193]
I notice that the last non-prerelease version on nuget is 1.0.0, released 1.5 years ago. All releases since then have been prereleases (e.g. 1.1.0-CI00252)

Is there a reason you've only been using prereleases for so long?

**Comment by wallymathieu** [comment]
Could be nice with a 1.1.0 release.

**Comment by gusty** [comment]
AFAIK the only reason is stability.
We want to make sure that what goes in a 1.1.0 release doesn't break anything nor is reverted later.
Pre-releases shouldn't pose any problem when a feature is added or a bug fixed.
Having said that, I personally would like to see a 1.1.0 before the end of the year.

**Comment by cmeeren** [comment]
Just posting a friendly reminder of this. Will we see 1.1.0 soon? :)

**Comment by wallymathieu** [comment]
You mean as a Christmas present 🎅 ?

**Comment by cmeeren** [comment]
Ho-ho-ho! 🦌🦌🦌🛷🎅

**Comment by gusty** [comment]
Thanks for the reminder. Precisely this week we (the maintainers) had a discussion regarding this.

I'll post here to share with everybody.

There are some pending PRs we wanted to include but we found some problems that would affect #192 and fixing these problems would require a small breaking change.

As we take seriously semantic versioning to avoid causing problems to people using this lib in production, we decided that we'll do those changes in a future (hopefully not far future) v2.0 but at the sametime we decided not to skip the long awaited v1.1

At the moment the way to go is to finish and merge #185 and possibly a preview of #192  while other features big features will be integrated in a v2.0 release.

We're open to suggestions and hope to get the 1.1 before the end of the year, if not for January, a new pre-release will follow soon and if everything is fine it will become 1.1
